2025.2/config.json

```diff
@@ -0,0 +1,60 @@
+{
+  "defaultFiles": "./defaultFiles",
```

Comment:
I have the CLI downloading the `defaultFiles` directory automatically, so anything we want to get auto downloaded can go in there.